style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  font-family: Arial, sans-serif;

```

Comment:
u can add background-size: cover; to avoid background repetition when the background size is smaller than the screen size